opentelemetry-http/src/lib.rs

```diff
@@ -125,7 +125,7 @@ mod isahc {
     }
 }
 
-#[cfg(any(feature = "hyper", feature = "hyper_tls"))]
```

Comment:
Do you know what this feature was introduced for?

Reply:
Got added in #853. Probably a mistake -  the feature was used without defining it in Cargo.toml. The latest rust toolchain seems to catch this now for error.,